crates/proposer-client/src/lib.rs

```diff
@@ -85,14 +85,14 @@ where
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub struct ProposerRequest {
+pub struct FEPProposerRequest {
```

Comment:
```suggestion
pub struct FepProposerRequest {
```
https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case